src/scss/bootpresso.scss

```diff
@@ -1,7 +1,9 @@
 // customizing
-@import "variables.scss";
+@import "variables.sass";
+@import "typography.sass";
+@import "vendor/dripicons";
```

Comment:
vendor/dripicons should not be in the folder src/scss but in a separate src/vendor folder, and than maybe refer to it like so `@import "@/src/vendor/dripicons";`